src/main/target/ANYFCF7/target.c

```diff
@@ -40,6 +36,11 @@ timerHardware_t timerHardware[] = {
     DEF_TIM(TIM5,  CH1, PA0,  TIM_USE_OUTPUT_AUTO,               0, 0 ), // S7_OUT DMA1_ST2
     DEF_TIM(TIM2,  CH2, PB3,  TIM_USE_OUTPUT_AUTO | TIM_USE_LED, 0, 0 ), // S8_OUT DMA1_ST6
     DEF_TIM(TIM3,  CH1, PB4,  TIM_USE_OUTPUT_AUTO,               0, 0 ), // S9_OUT DMA1_ST4
+    
```

Comment:
This will change the order of the outputs, but the comments are already all over the place...

Reply:
But labels on the comments seem to be all over the place, as S10 is before S6.

S3_IN used to be S1 in INAV.